app/src/main/java/org/simple/clinic/appupdate/UpdateManager.kt

```diff
@@ -0,0 +1,40 @@
+package org.simple.clinic.appupdate
+
+import com.google.android.play.core.appupdate.AppUpdateInfo
+import com.google.android.play.core.appupdate.AppUpdateManager
+import com.google.android.play.core.install.model.AppUpdateType.FLEXIBLE
+import com.google.android.play.core.install.model.UpdateAvailability.UPDATE_AVAILABLE
+import io.reactivex.Observable
+import javax.inject.Inject
+
+interface UpdateManager {
+  fun updateInfo(): Observable<UpdateInfo>
+}
+
+class PlayUpdateManager @Inject constructor(
```

Comment:
Is there a reason this implementation is in the same file? Can we break it out into another?

Reply:
No particular reason, I think it was because there was no other implementation for `UpdateManager` I have decided to keep both of them in the same file.

Reply:
Right, for interfaces and their class implementations I generally prefer to have them as separate files even if there's only one implementation. But that's a personal preference, maybe we can decide on a convention later.